src/panels/flow-histogram-react/FlowHistogramHelpers.ts

```diff
@@ -1,96 +1,217 @@
-import { SelectableValue } from "@grafana/data";
-import { FlowHistogramOptionsProps } from "./FlowHistogramTypes";
-
-const formatFlowHistogram = (histogramUnit: SelectableValue<string>, v) => {
-    let divisor = 1;
-
-    if (histogramUnit.label === 'KB') {
-        divisor = 1000
-    } else if (histogramUnit.label === 'MB') {
-        divisor = 1000000
-    } else if (histogramUnit.label === 'GB') {
-        divisor = 1000000000;
-    }
+/* eslint-disable no-restricted-imports */
```

Comment:
Nit: I'm not sure what eslint rule you're disabling here, but you should probably leave a little comment as to why you're working around the linting tool ( or maybe disable the rule ).